arches_component_lab/src/arches_component_lab/constants.ts

```diff
@@ -1,3 +0,0 @@
-export const PREF_LABEL = "prefLabel";
```

Comment:
same (probably a poor decision on my part to put these here, controlled-lists seems better, how many apps care about labels?)

Reply:
yep, worth re-declaring them there I think